### PR TITLE
got alg will work on code after lab

### PR DESCRIPTION
**Pull request by M-Ravenmoore**
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/linkedList/README.md [hunk 1]
```diff
@@ -19,4 +19,5 @@ i will update this asap
 <!-- Embedded whiteboard image -->
 ![whiteboard](../assets/CC5-whiteboard.jpg)
 ![whiteboard](../assets/CC6-whiteboard.jpg)
-![whiteboard](../assets/CC7-whiteboard.png)
\ No newline at end of file
+![whiteboard](../assets/CC7-whiteboard.png)
+![whiteboard](../assets/CC8-whiteboard.jpg)
```
Comment (adrianhuebner) [inline javascript/code-challenges/linkedList/README.md:23]: Great start with the whiteboard, would love to see code implemented and tests added!
